README.md

```diff
@@ -1,2 +1,44 @@
 # git-filter-repo-docker-action
 A GitHub Docker Action to run git-filter-repo and push the result to a GitHub repository using ssh
+
+This repo contains a github action to trigger a custom git-repo-filter command.
+
+It is designed to be run on pushes to the main or master branch of a repository
+and to then push the resulting filtered repo to another remote using an ssh key.
+
+# Example Workflow
+```
+name: CI
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  filter:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: git-filter-repo
+      id: filter
```

Comment:
Is this used anywhere? I’m not sure if I understand the documentation about this ID field correctly.

Reply:
I think it is just used if we want to reference "the step" 

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsid

I'm not sure if it could actually be omitted

Reply:
Tested and it's not needed